src/js/providers/bandwidth-monitor.ts

```diff
@@ -1,11 +1,12 @@
 import { BANDWIDTH_ESTIMATE } from 'events/events';
 import { isValidNumber } from 'utils/underscore';
+import { DefaultProvider } from 'types/generic.type';
 
-export default function BandwidthMonitor(provider, initialEstimate) {
-    let bandwidthMonitorInterval = null;
+export default function BandwidthMonitor(provider: DefaultProvider, initialEstimate: number): BandwidthMonitor {
```

Comment:
Added this `DefaultProvider` type because it looks like only shaka and hlsjs providers use this.

Reply:
we should make a note in the TS confluence doc about moving `DefaultProvider` to `providers/default` once that gets typed

Reply:
👍 I also added a note for these in the TS spreadsheet by the files